cockatrice/src/game/player/player.h

```diff
@@ -268,8 +268,9 @@ private slots:
 
     bool movingCardsUntil;
     QTimer *moveTopCardTimer;
-    QString previousMovingCardsUntilExpr = {};
-    int previousMovingCardsUntilNumberOfHits = 1;
+    QString movingCardsUntilExpr = {};
+    int movingCardsUntilNumberOfHits = 1;
+    bool movingCardsUntilAutoPlay = false;
```

Comment:
do we want this to be a setting users can persist?

Reply:
the others also don't persist though